kyuubi-server/src/main/scala/org/apache/kyuubi/engine/flink/FlinkProcessBuilder.scala

```diff
@@ -104,13 +104,23 @@ class FlinkProcessBuilder(
         val userJars = conf.get(ENGINE_FLINK_APPLICATION_JARS)
         userJars.foreach(jars => flinkExtraJars ++= jars.split(","))
 
+        val hiveConfDirOpt = env.get("HIVE_CONF_DIR")
+        hiveConfDirOpt.foreach { hiveConfDir =>
+          val hiveConf = s"${hiveConfDir}${File.separator}hive-site.xml"
```

Comment:
Would it be better to check if the hive-site.xml file exists here and throw an exception if it doesn't? @hadoopkandy WDYT?

Reply:
i agree